01_reverse.rb

```diff
@@ -15,5 +15,16 @@
 #
 
 def reverse (string)
-  # Your code here
+  (string).reverse
```

Comment:
👍 It works, the brackets around `string` aren't required.
`string.reverse` will also work.